pilot/pkg/model/push_context.go

```diff
@@ -904,6 +904,24 @@ func (ps *PushContext) ServiceForHostname(proxy *Proxy, hostname host.Name) *Ser
 	return nil
 }
 
+// ServicesForHostname returns a list of services that fall under the hostname provided. This hostname
+// can be a wildcard.
+func (ps *PushContext) ServicesForHostname(proxy *Proxy, hostname host.Name, namespace string) []*Service {
```

Comment:
`namespace` is not used which is suspicious

Reply:
I think namespace is not needed here?

Reply:
It is not needed I think

Reply:
yeah lgtm to remove since its already using SidecarScope which handles the ns part.